OsLogin/src/V1beta/Gapic/OsLoginServiceGapicClient.php

```diff
@@ -29,14 +29,14 @@
 use Google\ApiCore\ApiException;
 use Google\ApiCore\CredentialsWrapper;
 use Google\ApiCore\GapicClientTrait;
-
 use Google\ApiCore\PathTemplate;
 use Google\ApiCore\RequestParamsHeaderDescriptor;
 use Google\ApiCore\RetrySettings;
 use Google\ApiCore\Transport\TransportInterface;
 use Google\ApiCore\ValidationException;
 use Google\Auth\FetchAuthTokenInterface;
 use Google\Cloud\OsLogin\Common\SshPublicKey;
+use Google\Cloud\OsLogin\V1beta\CreateSshPublicKeyRequest;
```

Comment:
Interesting, this class wasn't added to this PR even though it's a part of the [oslogin.proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/oslogin/v1/oslogin.proto#L127)

Reply:
Looking at the commits, the file `CreateSshPublicKeyRequest.php` is added [in the generator commit](https://github.com/googleapis/google-cloud-php/pull/5612/commits/661c6855ee9c782f74d88a2a97f580a8f863a441#diff-03a4776db905af21602878e8bf859917447eeca63177952b64bf97670a73ee94), but then doesn't get added [in the owlbot commit](https://github.com/googleapis/google-cloud-php/pull/5612/commits/eccbd16020c64ce22e11a5677e946ce782eb8584)

Reply:
I think it's possible owlbot is misconfigured, as there hasn't been an update like this to OsLogin since we migrated it to owbot.

Reply:
fixed in https://github.com/googleapis/google-cloud-php/pull/5678!